tmexp/preprocess.py

```diff
@@ -322,12 +333,15 @@ def feature_extractor(uast_obj: Any) -> Iterator[Tuple[str, str]]:
                 words = [w for w in word.split() if w.lower() not in stop_words]
             else:
                 words = [word]
-            words = [w for word in words for w in word.split("_")]
-            words = [
-                w
-                for word in words
-                for w in re.findall(r"[A-Z]?[a-z]+|[A-Z]+(?=[A-Z]|$)", word)
-            ]
+            if use_nn:
+                words = [token_parser.split(w) for w in words]
```

Comment:
![](https://i.kym-cdn.com/photos/images/newsfeed/000/534/153/f87.jpg)
```suggestion
                words = [w for word in words for w in token_parser.split(word)]
```
?